schemas/dto/jobExecutionLogDto.json

```diff
@@ -1,23 +0,0 @@
-{
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "Job execution log dto schema",
-  "type": "object",
-  "additionalProperties": false,
-  "properties": {
-    "jobExecutionId": {
-      "description": "UUID",
-      "$ref": "../../raml-util/schemas/uuid.schema"
-    },
-    "jobExecutionResultLogs": {
-      "description": "Set of action logs",
-      "type": "array",
-      "items": {
-        "$ref": "./actionLog.json"
```

Comment:
Hello @SatishkumarPurswani, please delete the jobExecutionLogDto.sample (sample file) and actionLog.json file as well

Reply:
Hi @RuslanLavrov - I have removed the mentioned files, Thanks for pointing it out, it was missed by me.

Reply:
thank you!